### ISSUE TITLE
nf-core module: entrezdirect/esearch 

### ISSUE THREAD

**Issue by alxndrdiaz** [sanger-tol/blobtoolkit#2]
Write [entrez-direct](https://www.ncbi.nlm.nih.gov/books/NBK179288/) nf-core module: entrezdirect/esearch;  nf-core modules issue: [new module: entrezdirect/esearch #1771](https://github.com/nf-core/modules/issues/1771).

**Comment by alxndrdiaz** [comment]
Latest entrez-direct version: [16.2-0 Bioconda Package Index](https://bioconda.github.io/recipes/entrez-direct/README.html#package-entrez-direct).

**Comment by alxndrdiaz** [comment]
Latest entrez-direct (16.2-1) Docker container version still has a bug: 

```
entrezdirect_img_latest='quay.io/biocontainers/entrez-direct:16.2--he881be0_1'
database="pubmed"
term="Hox-genes"

docker run \
--rm \
-v $(pwd):$(pwd) \
-w $(pwd) \
-t ${entrezdirect_img_previous} \
esearch -db ${database} -query ${term}
```

```
Can't locate Time/HiRes.pm in @INC (you may need to install the Time::HiRes module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.28.1 /usr/local/share/perl/5.28.1 /usr/lib/x86_64-linux-gnu/perl5/5.28 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.28 /usr/share/perl/5.28 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base).
BEGIN failed--compilation aborted.

<ENTREZ_DIRECT>
  <Db>pubmed</Db>
  <WebEnv>MCID_62ac9c50d7198b20634e5a32</WebEnv>
  <QueryKey>1</QueryKey>
  <Count>17387</Count>
  <Step>1</Step>
</ENTREZ_DIRECT>
```

For development a previous working version (13.9-0) will be used.

**Comment by alxndrdiaz** [comment]
Pull request: [nf-core modules pull request #1807](https://github.com/nf-core/modules/pull/1807).

**Comment by alxndrdiaz** [comment]
PR merged: [new (fixed) module: entrezdirect/esearch #1927](https://github.com/nf-core/modules/pull/1927).
